### PR TITLE
Clubs and Moist Flesh blocks

### PR DESCRIPTION
**Pull request by Shnupbups**
Clubs are a new weapon! They're uncraftable (and will remain so!) but will be obtainable from Ogres and Trolls when they're added, as they will be their weapons of choice. They're *very* slow, but *very* powerful. You also can't use your off hand item whilst holding one. (it won't even render in first person!)
Also added Moist Flesh Block. Deliciously disgusting!
All of these have wonderful textures by the fantastic MoonlitLamps!

### CODE REVIEW

File: src/main/java/com/fabriccommunity/spookytime/mixin/client/FirstPersonRendererMixin.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+@Mixin(FirstPersonRenderer.class)
+public class FirstPersonRendererMixin {
+	@Final
+	@Shadow
+	private MinecraftClient client;
+	
+	@ModifyVariable(method = "renderFirstPersonItem(F)V", ordinal = 1, at =
+	@At(value = "INVOKE", target = "Lnet/minecraft/client/render/FirstPersonRenderer;rotate(FF)V"))
+	private boolean dontRenderOffHandItem(boolean bool) {
```
Comment (Shrimpstress) [inline src/main/java/com/fabriccommunity/spookytime/mixin/client/FirstPersonRendererMixin.java]: Change bool to prevBool.

**Review by Shrimpstress — CHANGES_REQUESTED** [review]
Everything looks good to me besides one little thing.

File: src/main/java/com/fabriccommunity/spookytime/mixin/client/FirstPersonRendererMixin.java (hunk 1 above)
Comment (Juuxel) [reply src/main/java/com/fabriccommunity/spookytime/mixin/client/FirstPersonRendererMixin.java]: Or `original`.

Comment (Shrimpstress) [reply src/main/java/com/fabriccommunity/spookytime/mixin/client/FirstPersonRendererMixin.java]: Yeah, either one of these would be acceptable.

**Review by Juuxel — APPROVED** [review]
Looks good apart from that one name that Zundrel pointed out.

**Review by Shrimpstress — APPROVED** [review]
Everything looks good to me now.

**Pull request merged**